src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cookies.cs

```diff
@@ -585,6 +710,18 @@ public async Task GetAsyncWithBasicAuth_ReceiveSetCookie_CookieSent()
             });
         }
 
+        // Workaround for Framework, where CookieCollection does not implement IEnumerable<Cookie>
```

Comment:
Why do we have this method? Comment says that it's workaround for Full FW, but it's only used in tests which do not run for Full FW.
Or is it compilation problem? If so, I'd mention that in the comment.

Reply:
And `CookieCollection` has a public [indexer](https://referencesource.microsoft.com/#System/net/System/Net/cookiecollection.cs,6da9e8a0fe5eb89e,references) in both: Full FW and Core. But you probably already tried that and it didn't work for you.

Reply:
> but it's only used in tests which do not run for Full FW

This is shared code, tests are also being executed for `System.Net.Http.WinHttpClient`, which includes a 4.7.2 Framework run.

> And CookieCollection has a public indexer in both: Full FW and Core. But you probably already tried that and it didn't work for you.

I thought indexers didn't exist back in the stone age, so didn't even try 😅.